buttercup/cogs/history.py

```diff
@@ -377,7 +377,7 @@ def get_all_rate_data(
             # Convert date strings to datetime objects
             new_frame["date"] = new_frame["date"].apply(lambda x: parser.parse(x))
             # Add the data to the list
-            rate_data = rate_data.append(new_frame.set_index("date"))
+            rate_data = pd.concat([rate_data, new_frame.set_index("date")])

```

Comment:
I looked at the difference between .concat vs .append. Is there a reason why you chose to switch in this case?

Reply:
I got a warning that `.append` is deprecated and might not work in future versions of `pandas`, so I chose to fix it in this PR